### ISSUE TITLE
"The following hardware was not installed"

### ISSUE THREAD

**Issue by Hmialia** [itsmikethetech/Virtual-Display-Driver#117]
Hi, I'm trying to install this driver, but it doesn't seem to be accepted by the system, possibly due to certain system policies.
![image](https://github.com/itsmikethetech/Virtual-Display-Driver/assets/118377714/24150c19-6320-4090-9821-2dc2f82c5910)

First, I ran "installCert.bat" as an administrator, and the command prompt output indicated everything was fine. However, when I tried to add the hardware in Device Manager, the system gave an "An error occurred during the installation of the device." message at the final step. I'm at a loss on how to resolve this issue.
![image](https://github.com/itsmikethetech/Virtual-Display-Driver/assets/118377714/a3bda5c9-3028-4b95-9a46-1094540d56c0)

Additionally, my system drive letter is "X" instead of "C". I understand this is non-standard, but I had to do this for certain reasons. I'm unsure if this issue is related to the drive letter.

If you're willing to help, I can provide more information as needed. I sincerely appreciate any assistance you can offer.

**Comment by bud3699** [comment]
is windows installer service started in services ???

windows + r
services.msc

then scroll down to Windows Installer service and check if its running, if not, right click then click start

**Comment by bud3699** [comment]
However, i do believe that your system drive being x will cause issues in this version of the virtual display driver as it will not match the required path for options.txt

**Comment by Hmialia** [comment]
Thank you for the method you provided. I found that my "Windows Installer service" is set to stop by default, and I need to manually start it  each time. Unfortunately, I am still encountering the same issue.

Regarding my system drive letter issue, I had already copied the "IddSampleDriver" folder to both the C: drive and the X: drive, but this did not resolve the problem.

I think that changing the method of placing the folder from manual copying to a fixed location to embedding it within the program's internal directory might be a better approach, and I have seen other issues mentioning this method as well. In any case, I really appreciate your assistance.
